.github/workflows/depsreview.yaml

```diff
@@ -0,0 +1,14 @@
+name: 'Dependency Review'
+on: [pull_request]
+
+permissions:
+  contents: read
+
+jobs:
+  dependency-review:
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Checkout Repository'
+        uses: actions/checkout@v3
+      - name: 'Dependency Review'
+        uses: actions/dependency-review-action@v1
```

Comment:
could you update it to v2 ?

Reply:
did the update and prettier, could be merged @rejas (we can add changelog later with another PR)

Reply:
so you can change the PR but not add an entry?

Reply:
did this in the online editor, could edit the file but did not find so far how to add another file ...

Reply:
Maybe a quick PR with changelog entry and some dependency updates :-) ?

Reply:
https://github.com/MichMich/MagicMirror/pull/2960 :-)